runtime/oti/jvminit.h

```diff
@@ -453,6 +453,9 @@ enum INIT_STAGE {
 #define VMOPT_XXTRANSPARENT_HUGEPAGE "-XX:+TransparentHugePage"
 #define VMOPT_XXNOTRANSPARENT_HUGEPAGE "-XX:-TransparentHugePage"
 
+#define VMOPT_XXDEEP_SCAN "-XX:+DeepScan"
+#define VMOPT_XXNODEEP_SCAN "-XX:-DeepScan"
```

Comment:
Can this have a more explicit name?  Think of it as the user interface to the feature and pick a name that's both unique and likely to be meaningful to the user.